pkg/controller/account/account_controller.go

```diff
@@ -234,11 +230,6 @@ type SREConsoleInput struct {
 	SecretName, NameSpace, awsCredsConsoleLoginURL string
 }
 
-// input for new aws client
-type newAwsClientInput struct {
-	awsCredsSecretIDKey, awsCredsSecretAccessKey, awsToken, awsRegion, secretName, nameSpace string
```

Comment:
Looking at this the first time around I thought it might fit better in the iam.go file but now that I'm looking at it in the diff I'm not sure.  Thoughts?

Reply:
This should stay with the main controller logic.